tests/download.spec.ts

```diff
@@ -580,3 +581,65 @@ it.describe('download event', () => {
     await page.close();
   });
 });
+
+it('should be able to download a PDF file', async ({ browser, server, asset }) => {
+  const page = await browser.newPage({ acceptDownloads: true });
+  await page.goto(server.EMPTY_PAGE);
+  await page.setContent(`
+    <a href="/empty.pdf" download>download</a>
+  `);
+  const [download] = await Promise.all([
+    page.waitForEvent('download'),
+    page.click('a'),
+  ]);
+  await assertDownloadToPDF(download, asset('empty.pdf'));
+  await page.close();
+});
+
+it('should be able to download a inline PDF file', async ({ browser, server, asset }) => {
+  it.fixme(true, 'Wrong download content gets returned');
+  const page = await browser.newPage({ acceptDownloads: true });
+  await page.goto(server.EMPTY_PAGE);
+  await page.route('**/empty.pdf', async route => {
+    // @ts-expect-error
+    const response = await route._continueToResponse({});
```

Comment:
Given that we may drop this response interception API I'd rather you set Content-Disposition in server.setRoute

Reply:
discussed offline, using request.get for now.